src/TesterInternal/General/BuiltInSerializerTests.cs

```diff
@@ -29,7 +29,7 @@ public class BuiltInSerializerTests
         public void InitializeForTesting()
         {
             TraceLogger.Initialize(new NodeConfiguration());
-            SerializationManager.Initialize(false, null);
+            SerializationManager.Initialize(false, null, false);
```

Comment:
It does convert int back to int. In the test <code>long</code> was casted to <code>int</code>.